man/standard-specifiers.xml

```diff
@@ -34,6 +34,11 @@
     <entry>Short host name</entry>
     <entry>The hostname of the running system, truncated at the first dot to remove any domain component.</entry>
   </row>
+  <row id='R'>
+    <entry><literal>%R</literal></entry>
+    <entry>Pretty host name</entry>
+    <entry>The pretty hostname of the running system. If not set, resolves to the short hostname.</entry>
```

Comment:
maybe add a comment that this is PRETTY_HOSTNAME from machine-info(5).

Reply:
Makes sense, updated.